src/pages/Option/Index.js

```diff
@@ -55,35 +55,38 @@ export function getPosition(rawPositionData,symbolData) {
   let positionQuery = []
   for(let i = 0; i < rawPositionData.positions.length; i++){
     const temp = rawPositionData.positions[i]
-    const markPrice = symbolData.find(obj => {
-      return obj.address === temp[0]
-    }).markPrice
-    const initialMarginRatio = symbolData.find(obj => {
-      return obj.address === temp[0]
-    }).initialMarginRatio
-    const indexPrice = symbolData.find(obj => {
-      return obj.address === temp[0]
-    }).indexPrice
-    const marginUsage = Math.abs(temp[2].toNumber() * indexPrice) * initialMarginRatio
-    const unrealizedPnl = temp[2].toNumber() * indexPrice - temp.cost
-    const position = {
-      symbol: temp[1],
-      address: temp[0],
-      position: temp[2].toNumber(),
-      entryPrice: safeDiv(temp.cost,temp[2]),
-      markPrice: symbolData.find(obj => {
+    const volume = ethers.utils.formatUnits(temp[2], 18)
+    if (volume!=0){
+
+      const markPrice = symbolData.find(obj => {
         return obj.address === temp[0]
-      }).markPrice,
-      marginUsage: symbolData.find(obj => {
+      }).markPrice
+      const initialMarginRatio = symbolData.find(obj => {
         return obj.address === temp[0]
-      }).initialMarginRatio,
-      unrealizedPnl: symbolData.find(obj => {
+      }).initialMarginRatio
+      const indexPrice = symbolData.find(obj => {
         return obj.address === temp[0]
```

Comment:
refactor idea: filter for object which its `address === temp[0]`, and destructure its `markPrice, initialMarginRatio, indexPrice`